policyengine_canada/variables/household/income/individual/pension_income.py

```diff
@@ -0,0 +1,10 @@
+from policyengine_canada.model_api import *
+
+
+class pension_income(Variable):
+    value_type = float
+    entity = Person
+    label = "Pension Income"
+    unit = CAD
+    documentation = "Income from the canada pension plan"
```

Comment:
What about private pension income?
```suggestion
    documentation = "Income from the Canada pension plan"
```

Reply:
Should we use the pension and savings plan income variable instead?

Reply:
If that's what the law refers to ;)

Reply:
Found it! Absolutely right